templates/vertical-interactive-map/page.html.hbs

```diff
@@ -17,10 +17,10 @@
     {{> templates/vertical-interactive-map/script/locationbias modifier="main" }}
     {{> templates/vertical-interactive-map/script/locationbias modifier="mobileMap" }}
   {{/script/core }}
-  <script src="{{relativePath}}/locator-components.js" defer></script>
+  <script src="{{relativePath}}/locator-bundle.js" defer></script>
     <div class="Answers AnswersVerticalMap CollapsibleFilters InteractiveMap InteractiveMap--listShown js-answersInteractiveMap">
       <div class="Answers-content">
-        <div class="Answers-contentWrap js-locator-contentWrap">
+        <div class="Answers-contentWrap js-locator-contentWrap"  role="region" aria-label="Main location search">
```

Comment:
does this need to be multilang?

Reply:
ah, good catch

Reply:
I wrapped these in calls to the translate helper. Once the strings are approved in this PR I can send them to Tom to get translated